### ISSUE TITLE
Uncaught Error: stream.push() after EOF

### ISSUE THREAD

**Issue by olizilla** [ipfs-shipyard/ipfs-webui#1010]
**Describe the bug**
Uploading a file > 1MB fails, with the error:

```
Uncaught Error: stream.push() after EOF
    at new NodeError (errors-browser.js:30)
    at readableAddChunk (_stream_readable.js:269)
    at PullDuplexStream.push../node_modules/readable-stream/lib/_stream_readable.js.Readable.push (_stream_readable.js:240)
    at next (index.js:55)
    at FileReader.loaded (index.js:22)
```

I think the size is only relevent in so far as it is large enoug that it needs to be split into chunks. I can't recreate the issue with smaller files.

**of note, I do not get this error uploading the same ~1MB flie to the 2.4.4 release that is in the current stable IPFS Dektop 0.7 release.**

**To Reproduce**
1. Using the lastest build on webui.ipfs.io or https://github.com/ipfs-shipyard/ipfs-webui/commit/ee3764d87ea2262007651b733588a0968150bcfc
2. Upload an ~80MB file
3. See an error in the logs. the file is not added to the MFS dir.

**Expected behavior**
File should be added to the dir

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser FF and Chrome

**Comment by olizilla** [comment]
@hugomrdias do you know what changed in `ipfs-http-client` recently that might have triggered this?

**Comment by olizilla** [comment]
Moar stack

  | fn | @ | src |
  |-- | -- | -- | 
  | NodeError | @ | errors-browser.js:30
  | readableAddChunk | @ | _stream_readable.js:269
  | push../node_modules/readable-stream/lib/_stream_readable.js.Readable.push | @ | _stream_readable.js:240
  | next | @ | index.js:55
  | loaded | @ | index.js:22
  | FileReader (async) |   |  
  | (anonymous) | @ | index.js:31
  | drainPull | @ | index.js:46
  | _read | @ | index.js:64
  | push../node_modules/readable-stream/lib/_stream_readable.js.Readable.read | @ | _stream_readable.js:440
  | maybeReadMore_ | @ | _stream_readable.js:551
  | push../node_modules/process/browser.js.Item.run | @ | browser.js:167
  | drainQueue | @ | browser.js:131
  | setTimeout (async) |   |  
  | runTimeout | @ | browser.js:43
  | push../node_modules/process/browser.js.process.nextTick | @ | browser.js:156
  | onwrite | @ | _stream_writable.js:455
  | WritableState.onwrite | @ | _stream_writable.js:158
  | push../node_modules/iso-stream-http/lib/request.js.ClientRequest._write | @ | request.js:117
  | doWrite | @ | _stream_writable.js:405
  | writeOrBuffer | @ | _stream_writable.js:394
  | push../node_modules/readable-stream/lib/_stream_writable.js.Writable.write | @ | _stream_writable.js:303
  | ondata | @ | _stream_readable.js:640
  | emit | @ | events.js:136
  | addChunk | @ | _stream_readable.js:297
  | readableAddChunk | @ | _stream_readable.js:279
  | push../node_modules/readable-stream/lib/_stream_readable.js.Readable.push | @ | _stream_readable.js:240
  | push../node_modules/readable-stream/lib/_stream_transform.js.Transform.push | @ | _stream_transform.js:139
  | (anonymous) | @ | multipart.js:135
  | emit | @ | events.js:136
  | addChunk | @ | _stream_readable.js:297
  | readableAddChunk | @ | _stream_readable.js:279
  | push../node_modules/readable-stream/lib/_stream_readable.js.Readable.push | @ | _stream_readable.js:240
  | next | @ | index.js:55
  | loaded | @ | index.js:22
  | FileReader (async) |   |  
  | (anonymous) | @ | index.js:31
  | drainPull | @ | index.js:46
  | _read | @ | index.js:64
  | push../node_modules/readable-stream/lib/_stream_readable.js.Readable.read | @ | _stream_readable.js:440
  | resume_ | @ | _stream_readable.js:872
  | push../node_modules/process/browser.js.Item.run | @ | browser.js:167
  | drainQueue | @ | browser.js:131
  | setTimeout (async) |   |  
  | runTimeout | @ | browser.js:43
  | push../node_modules/process/browser.js.process.nextTick | @ | browser.js:156
  | resume | @ | _stream_readable.js:864
  | push../node_modules/readable-stream/lib/_stream_readable.js.Readable.resume | @ | _stream_readable.js:854
  | push../node_modules/readable-stream/lib/_stream_readable.js.Readable.on | @ | _stream_readable.js:772
  | push../node_modules/readable-stream/lib/_stream_readable.js.Readable.pipe | @ | _stream_readable.js:636
  | (anonymous) | @ | send-files-stream.js:157
  | (anonymous) | @ | add.js:59
  | (anonymous) | @ | index.js:39
  | (anonymous) | @ | index.js:34
  | (anonymous) | @ | add.js:81
  | _callee4$ | @ | files.js:248

**Comment by hacdias** [comment]
@olizilla perhaps this can be closed?

**Comment by lidel** [comment]
@olizilla I checked this recently on companion end, seems that fix got bubbled up and was released in v2.4.5:

- https://github.com/ipfs-shipyard/ipfs-webui/blob/v2.4.5/package.json#L44
  - https://github.com/ipfs-shipyard/ipfs-redux-bundle/blob/v5.1.2/package.json#L26
    - https://github.com/ipfs/js-ipfs-http-client/releases/tag/v30.1.4

**Comment by olizilla** [comment]
Thanks! Yep, this fixed.
